tools/lambda-promtail/lambda-promtail/s3.go

```diff
@@ -46,8 +46,12 @@ var (
 	// result:  retrieveAccessRights
 	requestUrlEndpointRegex = regexp.MustCompile(`([^0-9/]*$)`)
 
-	// don't push load balancer logs with traffic to these endpoints to loki
+	// for all accounts, don't push load balancer logs with traffic to these endpoints to loki
 	skipEndpoints = map[string]struct{}{"external-health-check": {}, "healthz": {}, "readiness": {}, "healthy": {}, "metrics": {}, "readyz": {}}
+
+	// if the load balancer is in the 'logs' aws account, don't push load balancer logs with traffic to these endpoints to loki
+	skipLogsAccountEndpoints = map[string]struct{}{"push": {}, "prometheus": {}}
+	logsAccountId            = "529633446764"
```

Comment:
A task for later, but we should probably make this kind of stuff configurable as a regexp from an environment variable, so it doesn't have to be hardcoded.